modules/src/core/ics03_connection/msgs/conn_open_ack.rs

```diff
@@ -29,11 +29,8 @@ pub struct MsgConnectionOpenAck {
 impl MsgConnectionOpenAck {
     /// Getter for accessing the `consensus_height` field from this message. Returns the special
     /// value `Height(0)` if this field is not set.
```

Comment:
The comments like this need to be updated too.

Reply:
I didn't find any other comments that referred to "Height zero"

[4a9dca7](https://github.com/informalsystems/ibc-rs/pull/2346/commits/4a9dca7bf1fba72aa4f25806a9442616a07c1c22)